### PR TITLE
Followed best practices

### PR DESCRIPTION
**Pull request by Ckrielle**
Changed the directory of the images to match the recommended one, and put the images under a collapsible markdown tag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Instructions for Windows
   #3F0E40,#350d36,#1164A3,#FFFFFF,#350D36,#FFFFFF,#2BAC76,#CD2553,#350d36,#FFFFFF
   ``` 
   <details><summary>Click here for screenshot</summary>
-  ![Aubergine](./docs/theme-pictures/Aubergine.png)
+  ![Aubergine](./assets/images/Aubergine.png)
```
Comment (sachinh19) [inline README.md]: I checked the this branch on your fork and the images are not getting rendered as expected on the README. Can you double check?

Comment (Ckrielle) [reply README.md]: Yes I saw, so far from researching I haven't found any documentation on images being capable to be put inside a collapsible markdown tag. I will ask on stackoverflow and inform you

Comment (Ckrielle) [reply README.md]: Turns out all works well if you use the html img tag. Gonna write the changes

Comment (sachinh19) [reply README.md]: Cool. Thanks for checking this.

**Review by sachinh19 — APPROVED** [review]

**Pull request merged**